src/components/Thanks/ShareStepper.vue

```diff
@@ -17,6 +17,10 @@
 				<span v-if="commentsMode">
 					<span class="data-hj-suppress">{{ lenderName }}</span>, go the extra mile
 				</span>
+				<span v-else-if="isFirstLoan">
+					<!-- eslint-disable-next-line max-len -->
+					<span class="data-hj-suppress">{{ lenderName }}</span>, You’re support means the world to <span class="data-hj-suppress">{{ loanName }}</span>. Let us help you change another life.
```

Comment:
Does `loanName` need to be suppressed?

Reply:
Commonly this is the borrower name, should be?

Reply:
It does look like we've hidden the loan name elswhere.